includes/Classifai/Admin/Notifications.php

```diff
@@ -48,10 +48,25 @@ public function maybe_render_notices() {
 
 		$needs_setup = get_transient( 'classifai_activation_notice' );
 		if ( $needs_setup ) {
-			printf(
-				'<div class="notice notice-warning"><p><a href="%s">' . esc_html__( 'ClassifAI requires setup', 'classifai' ) . '</a></p></div>',
-				esc_url( admin_url( 'admin.php?page=classifai_settings' ) )
-			);
+			if ( Onboarding::is_onboarding_completed() ) {
+				delete_transient( 'classifai_activation_notice' );
+				return;
+			}
+			?>
+			<div data-notice="plugin-activation" class="notice notice-success is-dismissible">
+				<div id="classifai-activation-notice">
+					<div class="classifai-logo">
+						<img src="<?php echo esc_url( CLASSIFAI_PLUGIN_URL . 'assets/img/classifai.png' ); ?>" alt="ClassifAI" />
```

Comment:
escaping function missing on alt attribute

Reply:
Because we are just outputting a hardcoded string (no translation or anything) there is no need to escape this